google/cloud/pubsub/pull_ack_handler.h

```diff
@@ -122,6 +123,14 @@ class PullAckHandler {
     }
     /// The implementation for `PullAckHandler::delivery_attempt()`
     virtual std::int32_t delivery_attempt() const { return 0; }
+    /// Returns the ack_id for the handler. No public interface for
+    /// `PullAckHandler`.
```

Comment:
Remove "No public interface for `PullAckHandler`." ? I don't know what that means.

Reply:
All the other functions are     /// The implementation for `PullAckHandler::delivery_attempt()`.

This makes it clear it intentionally does not have a public interface.

i.e.

```
class PullAckHandler {
 public:
    std::string ack_id() const { return ""; }

  }
```

Reply:
Maybe this is more clear

```
    /// Returns the subscription for the handler. There is no corresponding public
    /// interface to access the subscription id in `PullAckHandler`. This is intended to be used internally.
```

Reply:
Yeah, I like that more. I was about to suggest that you include the word "corresponding", because that is what confused me.

Reply:
Ack, done.